Simple Icon File Maker/Simple Icon File Maker/MainWindow.xaml

```diff
@@ -8,190 +8,206 @@
     xmlns:models="using:Simple_Icon_File_Maker.Models"
     mc:Ignorable="d">
 
-    <Grid>
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="360" />
-            <ColumnDefinition Width="*" />
-        </Grid.ColumnDefinitions>
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="360" />
+                <ColumnDefinition Width="*" />
+            </Grid.ColumnDefinitions>
 
-        <Grid.RowDefinitions>
-            <RowDefinition Height="auto" />
-            <RowDefinition Height="*" />
-        </Grid.RowDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="auto" />
+                <RowDefinition Height="*" />
+            </Grid.RowDefinitions>
 
-        <!--  Right side:  -->
+            <!--  Right side:  -->
 
-        <Grid
+            <Grid
             Grid.RowSpan="2"
             Grid.Column="1"
             AllowDrop="True"
             Background="{ThemeResource SystemControlBaseLowAcrylicWindowBrush}"
             DragOver="Border_DragOver"
             Drop="Grid_Drop">
+
+                <!-- Error Info Bar -->
+
+            <InfoBar x:Name="errorInfoBar" Title="An Error Has Occured" Severity="Error">
+                <InfoBar.Resources>
+                    <Storyboard x:Name="closeInfoBarStoryboard">
+                        <ObjectAnimationUsingKeyFrames
+                            Storyboard.TargetName="errorInfoBar"
+                            Storyboard.TargetProperty="IsOpen"
+                            Duration="0:0:5">
+                            <DiscreteObjectKeyFrame KeyTime="0:0:5" Value="False"/>
```

Comment:
The auto-dismiss could stay on for longer than 5 seconds, maybe 20. I see below the user cannot try to load a different image until the `errorInfoBar` is dismissed, so maybe we remove that dependency and allow this to show for 20 seconds.